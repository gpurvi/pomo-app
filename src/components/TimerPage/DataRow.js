import React from 'react';

const DataRow = (props) => (
    <tr>
        <td>{props.firstCol}</td>
        <td>{props.secondCol}</td>
    </tr>
);

export default DataRow;