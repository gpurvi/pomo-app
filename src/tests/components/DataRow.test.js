import React from 'react';
import {shallow} from 'enzyme';
import DataRow from '../../components/TimerPage/DataRow';

test('renders DataRow correctly', () => {
    const wrapper = shallow(<DataRow firstCol="name" secondCol="age"/>);
    expect(wrapper).toMatchSnapshot();
});
