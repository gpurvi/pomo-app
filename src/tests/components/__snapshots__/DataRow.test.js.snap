// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DataRow correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DataRow
    firstCol="name"
    secondCol="age"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <td>
          name
</td>,
        <td>
          age
</td>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "name",
        },
        "ref": null,
        "rendered": "name",
        "type": "td",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "age",
        },
        "ref": null,
        "rendered": "age",
        "type": "td",
      },
    ],
    "type": "tr",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <td>
            name
</td>,
          <td>
            age
</td>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "name",
          },
          "ref": null,
          "rendered": "name",
          "type": "td",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "age",
          },
          "ref": null,
          "rendered": "age",
          "type": "td",
        },
      ],
      "type": "tr",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
